rest-countries-API/script.js

```diff
@@ -1,3 +1,28 @@
+const countriesContainer = document.querySelector('.countries-container')
 fetch("https://restcountries.com/v3.1/all")
-  .then((res) => res.json())
-  .then(console.log);
+  .then((resolve) => resolve.json())
+  .then((data) => {
+    data.forEach((country) => {
```

Comment:
The purpose of using forEach over here is to get all countries' details and then used DOM methods to create country cards and to display